### PR TITLE
Added Grade 2 suport and UITextView suport

### PR DESCRIPTION
**Pull request by 7imbrook**

### CODE REVIEW

File: AccessBraille/AccessBraille/ABBrailleReader.h [hunk 1]
```diff
@@ -10,11 +10,28 @@
...
+static NSString *const ABPrefixLevelSeven = @"000110";
+
 @interface ABBrailleReader : NSObject <ABTouchReciever>
 
 + (NSString *)brailleStringFromTouchIDs:(NSArray *)touchIDs;
+- (void)characterReceived:(NSString *)brailleString;
 - (id)initWithAudioTarget:(id)target selector:(SEL)selector;
 
+// Testing
+- (NSString *)proccessString:(NSString *)brailleString;
```
Comment (piperchester) [inline AccessBraille/AccessBraille/ABBrailleReader.h]: Only one 'c' in process.

File: AccessBraille/AccessBraille/ABBrailleReader.m [hunk 2]
```diff
@@ -8,23 +8,54 @@
 
 #import "ABBrailleReader.h"
 #import "ABKeyboard.h"
+#import "UITextView+simpleadd.h"
 
 @implementation ABBrailleReader {
     
-    NSDictionary *grad1Lookup;
```
Comment (piperchester) [inline AccessBraille/AccessBraille/ABBrailleReader.m:14]: Change to 'grade'.

File: AccessBraille/AccessBraille/ABBrailleReader.m [hunk 3]
```diff
@@ -8,23 +8,54 @@
...
+    // Look ups
+    NSDictionary *grad2Lookup;
```
Comment (piperchester) [inline AccessBraille/AccessBraille/ABBrailleReader.m]: Here too...

File: AccessBraille/AccessBraille/BrailleTyperController.m [hunk 4]
```diff
@@ -92,24 +94,18 @@ - (void)viewDidUnload {
...
 # pragma mark - Typing Methods
 
 - (void)characterTyped:(NSString *)character withInfo:(NSDictionary *)info {
-    if (![info[ABBackspaceReceived] boolValue]) {
-        [_TextDrawing appendToText:character];
-        [keyboard startSpeakingString:character];
-    } else {
-        [_TextDrawing removeCharacter];
-    }
+    NSLog(@"Typed: %@", character);
```
Comment (piperchester) [inline AccessBraille/AccessBraille/BrailleTyperController.m:104]: A wise man once told me, no logging in master.

File: AccessBraille/AccessBraille/ABBrailleReader.m (hunk 2 above)
Comment (7imbrook) [inline AccessBraille/AccessBraille/ABBrailleReader.m:14]: Piper, that's a deleted line....

**Pull request merged**